activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/events/handlers/TaskCreatedEventHandler.java

```diff
@@ -45,24 +44,29 @@ public TaskCreatedEventHandler(TaskRepository taskRepository, EntityManager enti
     public void handle(ProcessEngineEvent event) {
         TaskCreatedEvent taskCreatedEvent = (TaskCreatedEvent) event;
         Task task = taskCreatedEvent.getTask();
-        
-        // Get processInstance reference proxy without database query
-        ProcessInstance processInstance = entityManager
-                .getReference(ProcessInstance.class, taskCreatedEvent.getProcessInstanceId());
 
-        task.setApplicationName(event.getApplicationName());
-        // Associate task with parent reference
-        task.setProcessInstance(processInstance);
-        
+        // not a standalone task
+        if (taskCreatedEvent.getProcessInstanceId() != null) {
+            // Get processInstance reference proxy without database query
+            ProcessInstance processInstance = entityManager
+                    .getReference(ProcessInstance.class,
+                                  taskCreatedEvent.getProcessInstanceId());
+
+            task.setApplicationName(event.getApplicationName());
```

Comment:
The applicationName field as we've been using it so far is just the rb name. So we should be able to populate that for a standalone task too?